File: ios/Classes/message/entity/video/VideoMessageEntity.swift
//  Created by 蒋具宏 on 2020/3/15.
//  视频消息实体
public class VideoMessageEntity : AbstractMessageEntity{
   
    /// 视频信息
    var videoInfo : VideoInfo?;
    
    /// 缩略图信息
    var videoSnapshotInfo : VideoSnapshotInfo?;
    
    override init() {
        super.init(MessageNodeType.Video);
    }
}

File: android/src/main/java/top/huic/tencent_im_plugin/message/entity/video/VideoMessageEntity.java
package top.huic.tencent_im_plugin.message.entity.video;

import java.io.Serializable;

import top.huic.tencent_im_plugin.enums.MessageNodeType;
import top.huic.tencent_im_plugin.message.entity.AbstractMessageEntity;

/**
 * 视频消息实体
 *
 * @author 蒋具宏
 */
public class VideoMessageEntity extends AbstractMessageEntity implements Serializable {

    /**
     * 视频信息
     */
    private VideoInfo videoInfo;

    /**
     * 缩略图信息
     */
    private VideoSnapshotInfo videoSnapshotInfo;

    public VideoMessageEntity() {
        super(MessageNodeType.Video);
    }

    public VideoInfo getVideoInfo() {
        return videoInfo;
    }

    public void setVideoInfo(VideoInfo videoInfo) {
        this.videoInfo = videoInfo;
    }

    public VideoSnapshotInfo getVideoSnapshotInfo() {
        return videoSnapshotInfo;
    }

    public void setVideoSnapshotInfo(VideoSnapshotInfo videoSnapshotInfo) {
        this.videoSnapshotInfo = videoSnapshotInfo;
    }
}